lib/assessment.rb

```diff
@@ -28,6 +31,7 @@ def some_odd(numbers)
   # numbers enumerable are even
   def every_even(numbers)
     # Your code goes here
+    numbers.select{|number| !number.zero?}.all?(&:even?)
```

Comment:
I think zero counts as an even number, but it's not an unreasonable consideration. Full credit.